website/frontend/src/pages/OurProducts/AnalyticsPage.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import Header from '../../components/OurProducts/Header';
 import Page from '../Page';
 import HeaderImage from '../../assets/img/OurProducts/Analytics/analytics-header.png';
-import { useInitScrollTop } from 'utils/customHooks';
+import { useInitScrollTop } from 'utilities/customHooks';
 import Section1Image from '../../assets/img/OurProducts/Analytics/section-1.png';
 import Section1OverlapImage from '../../assets/img/OurProducts/Analytics/section-1-overlap.png';
```

Comment:
Thanks @FaithDaka for refactoring the path names. 

- This is brilliant and ensures easy readability and maintainability.
- In the future PRs, we can go HARD on "eliminating" more of long (../../../../../) paths imports and continue to embrace the now "cleaner" ones I am seeing.